modules/persistent-storage-csi-snapshots-controller-sidecar.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-snapshots.adoc
+
+[id="snapshots-controller-sidecar_{context}"]
+= CSI snapshot controller and sidecar
+
+{product-title} provides a snapshot controller that is deployed into the control plane. In addition, the csi-snapshotter sidecar is a helper container that is deployed along with the CSI driver for your CSI storage solution.
+
+The CSI snapshot controller and sidecar provide volume snapshotting through the {product-title} API. These external components run in the cluster.
```

Comment:
> OCP provides only "The CSI snapshot controller". The sidecar is provided by CSI driver vendor (which may be Red Hat, in case of OCS and other CSI drivers). The sidecar is installed during CSI driver installation, which is documented by the driver vendor.

Updated the doc to reflect functionality of sidecar. However, would it be better to remove sidecar info altogether as there are no specific instructions needed by user?

Reply:
The text looks OK now. As for removing sidecar info altogether, I leave it up to you. It is fine for me.

Reply:
Okay, I will leave it in the docs then.